src/pages/Split.tsx

```diff
@@ -2,22 +2,23 @@ import { IonButton, IonContent, IonHeader, IonItem, IonItemDivider, IonLabel, Io
 import { useContext } from 'react';
 import './Start.css';
 import { ChangePage, CurrentSplit, PageType, SplitState } from './types';
+import Dinero from "dinero.js";
 
 interface Props {
     state: SplitState
 }
 
 export type SplitStatus = {
-    left: number,
-    total_per_person: Map<number, number>,
+    left: Dinero.Dinero,
```

Comment:
I prefer importing the `Dinero` type direty and not need to use `Dinero.Dinero`. But maybe that doesn't work because you need the constructor?

Reply:
exactly, some idiosyncrasy with their type definitions